src/core/agent/services/multiSigService.ts

```diff
@@ -615,6 +617,46 @@ class MultiSigService extends AgentService {
     };
   }
 
+  async readyMultisigRotation(identifierId: string) {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifierId
+    );
+
+    const multiSig = await this.props.signifyClient
+      .identifiers()
+      .get(metadata.signifyName);
+
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(metadata?.signifyName);
+
+    const nextSequence = (Number(multiSig.state.s) + 1).toString();
+
+    const rmids = members?.rotation;
+
+    const rstates: any[] = [];
+    await Promise.allSettled(
+      rmids.map(async (rotation: any) => {
+        const aid = await this.props.signifyClient
+          .keyStates()
+          .query(rotation.aid, nextSequence);
+        if (aid.done) {
```

Comment:
We should call it `op` and not `aid`. Since it's an operation that is returned from `.query`.

We should also wait until the operation is done - since it might not be automatically done